app/views/users/two_factor_authentication_setup/index.html.slim

```diff
@@ -5,6 +5,9 @@ p.mt-tiny.mb0
   = t('devise.two_factor_authentication.otp_setup_html')
 = simple_form_for(@two_factor_setup_form,
     html: { autocomplete: 'off', role: 'form' },
+    data: {\
+      unsupported_area_codes: PhoneNumberCapabilities::VOICE_UNSUPPORTED_US_AREA_CODES.to_json,\
+    },
```

Comment:
@monfresh: Here's the line that makes slim-lint upset. An alternative I'd prefer to disabling the `IndentHash` cop is figuring out how to make this a bit shorter.